src/components/SearchInput/SearchInput.stories.tsx

```diff
@@ -27,7 +27,6 @@ export const DropdownExample = () => {
 
   useEffect(() => {
     window.addEventListener('keypress', keyPressHandler);
-
```

Comment:
lol wut

Reply:
really important to the core KPIs of the business